greenwave/policies.py

```diff
@@ -903,12 +905,24 @@ class Policy(SafeYAMLObject):
     source = None
 
     def validate(self):
+        if self.subject_type is None and not self.subject_types:
+            raise SafeYAMLError("No subject types provided")
+        if (
+            self.subject_type
+            and self.subject_types
+            and self.subject_types != [self.subject_type]
```

Comment:
Would just consider setting both `subject_types` and `subject_type` as an error without comparing if they are same or not.

Reply:
Ah, this is not needed anymore, I had to use the special `"*"` value for remote policy to keep the parsing and behavior backwards compatible. So this conditional should be: `if self.subject_type is not None and self.subject_types:`